### PR TITLE
Tutorial's Spanish translation with some fixes in English

### PR DESCRIPTION
**Pull request by paulacasas83**
## Summary by CodeRabbit

- **New Features**
	- Enhanced clarity and engagement in the murder mystery website content with improved descriptions and storytelling elements.
	- Refined how-to-play instructions for directness and accessibility in both English and Spanish.

- **Bug Fixes**
	- Corrected grammatical inaccuracies and improved phrasing for better readability in both English and Spanish content.

### CODE REVIEW

File: static/es/index.template.html [hunk 1]
```diff
@@ -194,26 +194,33 @@ <h3>🌡️ Sleuth-o-meter</h3>
...
 			<p>Cada pista revelada disminuirá tu puntaje. Ten en cuenta que el sitio web aún está en beta, por lo que este sistema de puntuación, la interfaz o el diseño están sujetos a cambios.
-				Siéntete libre de proporcionar comentarios <a class="link-dark" href="https://forms.gle/dvA4Wr8LiuHFmDZN7">aquí</a> o en <a
+				No dudes en enviarnos tus comentarios <a class="link-dark" href="https://forms.gle/dvA4Wr8LiuHFmDZN7">aquí</a> o en <a
 				class="link-dark" href="https://www.reddit.com/r/mystery_o_matic/">nuestro subreddit</a>.
 			</p>
 		</div>
 	</div>
 
 
-	<div id="how-to-play" style="display:none">
```
Comment (neuromancer) [inline static/es/index.template.html:204]: This needs to be reverted before merging

File: static/es/index.template.html [hunk 2]
```diff
@@ -194,26 +194,33 @@ <h3>🌡️ Sleuth-o-meter</h3>
...
+	<div id="how-to-play">
 		<div class="container mt-5">
-			<p>mystery-o-matic presenta un nuevo misterio cada día, ambientado en un lugar específico, donde cada sospechoso
-				actúa de forma independiente. Las pistas proporcionadas requieren deducciones espaciales
-				espaciales y temporales para resolver el caso.</p>
+			<p class="fst-italic"> La pandilla se reúne alrededor de un tomo polvoriento adornado con extravagantes letras doradas en la portada que dicen <b>"Mystery-o-Matic"</b>.</p>
+			<p class="fst-italic">
+				—¿Por qué quieren que leamos esto? —pregunta Alice mientras sopla el polvo de la tapa.
```
Comment (neuromancer) [inline static/es/index.template.html:209]: Use an html entity instead of unicode symbol (`—`)

Comment (neuromancer) [reply static/es/index.template.html:209]: [Actually..](https://stackoverflow.com/questions/9808098/html5-which-is-better-using-a-character-entity-vs-using-a-character-directly) :thinking:

Comment (paulacasas83) [reply static/es/index.template.html:208]: Ok, cuando termine todas las traducciones puedo pasarle el texto a ChatGPT a ver si me lo deja con guión o si me lo cambia por comillas, ya que es una regla que siempre confundo en español.

File: static/es/index.template.html [hunk 3]
```diff
@@ -194,26 +194,33 @@ <h3>🌡️ Sleuth-o-meter</h3>
...
 		</div>
 
 		<div class="container mt-5">
-			<h4>🎓 Entendiendo las pistas y haciendo deducciones</h4>
+			<h4>🎓 Comprender las pistas y hacer deducciones</h4>
 
 			<p>
-			En nuestro misterio de asesinato, los personajes pueden moverse entre habitaciones cada 15 minutos. Esto nos ayuda a seguir dónde estaban en diferentes momentos.
-			Sus declaraciones nos dicen dónde estaban a horas específicas.
+				En nuestro misterio de asesinato, los personajes se mueven de una habitación a otra cada 15 minutos, lo que nos permite rastrear su ubicación a lo largo del tiempo a través de sus declaraciones.
```
Comment (neuromancer) [inline static/es/index.template.html:226]: Check this tab

File: static/en/index.template.html [hunk 4]
```diff
@@ -183,7 +184,7 @@ <h3>🌡️ Sleuth-o-meter</h3>
 									<lu class="list-group">
 											<li class="list-group-item list-group-item-danger">Super Sleuth</li>
 											<li class="list-group-item list-group-item-warning">Seasoned P.I.</li>
-											<li class="list-group-item list-group-item-success"><i class="notranslate">Amateur Gumshoe</i></li>
+											<li class="list-group-item list-group-item-success"> class="notranslate">Amateur Gumshoe</li>
```
[the remark this answers is not in the document]
Comment (neuromancer) [reply static/en/index.template.html]: Actually, just removed `class="notranslate">` since it is invalid

**Pull request merged**